src/Microsoft.Framework.Runtime/DefaultHost.cs

```diff
@@ -192,22 +192,9 @@ private void Initialize(DefaultHostOptions options, IServiceProvider hostService
 
         private void AddRuntimeServiceBreadcrumb()
         {
-#if DNX451
-            var runtimeAssembly = typeof(Servicing.Breadcrumbs).Assembly;
-#else
-            var runtimeAssembly = typeof(Servicing.Breadcrumbs).GetTypeInfo().Assembly;
-#endif
-
-            var version = runtimeAssembly
-                ?.GetCustomAttribute<AssemblyInformationalVersionAttribute>()
-                ?.InformationalVersion;
-
-            if (!string.IsNullOrWhiteSpace(version))
-            {
-                var semanticVersion = new NuGet.SemanticVersion(version);
-                Servicing.Breadcrumbs.Instance.AddBreadcrumb(runtimeAssembly.GetName().Name, semanticVersion);
-            }
+            var env = ServiceProvider.GetService(typeof(IRuntimeEnvironment)) as IRuntimeEnvironment;
```

Comment:
nit: Consider using regular cast instead of `as` casting. `as` serves a different purpose.